### PR TITLE
Document HTML robot windows

### PR DESCRIPTION
**Pull request by omichel**
Relates to https://github.com/omichel/webots/pull/4001

### CODE REVIEW

**Comment by omichel** [comment]
:+1:

File: guide/controller-plugin.md [hunk 1]
```diff
@@ -301,3 +301,23 @@ A complete sample (communicating with the e-puck robot using bluetooth) can be
...
+
+A simple example of a HTML robot window is provided in the `robots/thymio/thymio2.wbt` sample simulation and demonstrates:
+
+- How to display sensor information in the robot window.
+- How to send user interface events (like mouse clicks) from the robot window to the controller program.
+- How to change the title of the robot window from the controller program.
+
+Currently, HTML robot windows can communicate only with C and C++ controller programs, but this will be extended soon to Python, Java and MATLAB controllers.
+
+The e-puck, Darwin-OP and generic robot window are still using the native robot window. However, they will soon be ported to the HTML robot window. After that, the native robot window system will be progressively phased out.
```
Comment (stefaniapedrazzi) [inline guide/controller-plugin.md]: I think that the first robot window (followed by ``are still using``) should be plural

**Comment by omichel** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
👍

File: guide/controller-plugin.md [hunk 2]
```diff
@@ -301,3 +301,23 @@ A complete sample (communicating with the e-puck robot using bluetooth) can be
 found in this directory:
 
 `WEBOTS_HOME/projects/robots/e-puck/plugins/remote_controls/e-puck_bluetooth`
+
+### HTML robot window
+
+As an alternative to native robot windows described earlier in this section, it is also possible to implement a HTML robot window. HTML robot windows present several advantages over the native robot windows:
+
+1. They are easier to implement: simple HTML layout, simple Javascript programming, no compilation needed.
```
Comment (fabienrohrer) [inline guide/controller-plugin.md]: When reading this doc, I have the feeling to read a pros list of using it over the old system (I'm speaking about the keywords `easy`, `very simple`, etc.) while a new user just want to read what system he should use and how to use it.
Why not to document this new system first, and the old system secondly, indicating in one sentence that the new system should be preferred?

**Comment by omichel** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

**Comment by fabienrohrer** [comment]
👍  (seems much more well integrated than the previous version, clear and concise)

**Pull request merged**